website/project/presentation.rst

```diff
@@ -84,175 +90,91 @@ simply working individually.
-   will make your presentation less awkward, because you’ll actually be
-   presenting to other people, and you will feel more obligated to stay
-   within the 15 minute limit (as opposed to getting into a “let’s do a
-   retake” mentality). Plus, the people from the other teams may give
-   you some valuable feedback at the end of the meeting (and will
-   themselves benefit from seeing how another team approaches giving a
-   presentation; you should also volunteer to attend other presentations
-   precisely for this reason!)
-3. If feasible, deliver the presentation again at a later time,
-   incorporating the feedback you received from the other students.
-
-All that said, you are allowed to record your presentation in other ways
-if you feel that will work better for you. However, if you’ve never
-prepared a pre-recorded presentation before, we suggest you use Zoom as
-described above.
-
-We suggest you allocate roughly 5 minutes for each portion of the
-presentation. Please note that the design/implementation and demo
-portions can be interwoven together, as opposed to treating them as
-completely distinct portions.
-
-Take into account that the presentation itself doesn’t have to be
-delivered by all team members. In fact, for a 15 minute presentation,
-you probably want no more than 2-3 people speaking in total. If you feel
-comfortable doing so, please make sure your camera is turned on when you
-speak during the presentation.
-
-Submission Instructions
------------------------
-
-You will submit your presentation through Panopto. On the CMSC 22000 canvas
-site, go to "Panopto Video", and then into the folder "Final Presentations [Submission Folder]".
-To upload your video, click on the "Create" button (which appears to the right
-of the search bar), and select "Upload media". You will then be prompted
-to upload a file. Please make sure the video file includes the name of
-your team in it.
+Live Demo
+---------
 
-Please note that this submission folder is set up so that only the person
-who uploads a video can see that video. Your teammates should not be alarmed
-if they go to this folder and do not see your team's video there.
+During the final exam slot for the class, you must run a demo of your
+work meeting the following requirements:
 
+- You must show a program in execution.
+- The program *must* use code you have written this quarter.
+- The code you run *must* be in the ``dev`` branch of the repository.
+  You cannot demo code that is still in one of your branches and has
+  not gone through a code review.
+- To ensure there is enough time for all the teams to showcase their
+  work, your demo should be around 5 minutes long, and never more
+  than 10 minutes long.
 
+We recommend that you think carefully about what you want to show
+in your demo, and that you plan ahead of time the exact steps you
+will take during the demo (as opposed to doing a more improvised demo).
 
-Q&A
```

Comment:
Will there be a brief Q&A during the live demo?

Reply:
There won't be a formal Q&A like previous years (that was "scored"), but I've clarified in the Live Demo section that we may ask some questions, but that they won't factor into their score.